src/avfoundation.cs

```diff
@@ -10868,4 +10868,131 @@ interface AVContentProposal : NSCopying
+		bool ShouldRetryContentKeyRequest (AVContentKeySession session, AVContentKeyRequest keyRequest, string retryReason);
+
+		[Export ("contentKeySessionContentProtectionSessionIdentifierDidChange:")]
+		void DidChange (AVContentKeySession session);
+	}
+
+	partial interface IAVContentKeySessionDelegate {}
+
+	[TV (10,2), Mac (10,12,4), iOS (10,3)]
+	[BaseType (typeof (NSObject))]
+	interface AVContentKeySession {
+		[Static]
+		[Export ("contentKeySessionWithKeySystem:storageDirectoryAtURL:")]
+		AVContentKeySession Create (string keySystem, [NullAllowed] NSUrl storageURL);
+
+		[Export ("setDelegate:queue:")]
+		void SetDelegate ([NullAllowed] IAVContentKeySessionDelegate newDelegate, [NullAllowed] DispatchQueue delegateQueue);
+
+		[Wrap ("WeakDelegate")]
+		[NullAllowed]
+		IAVContentKeySessionDelegate Delegate { get; }
+
+		[NullAllowed, Export ("delegate", ArgumentSemantic.Weak)]
+		NSObject WeakDelegate { get; }
+
+		[NullAllowed, Export ("delegateQueue")]
+		DispatchQueue DelegateQueue { get; }
+
+		[NullAllowed, Export ("storageURL")]
+		NSUrl StorageURL { get; }
+
+		[Export ("keySystem")]
+		string KeySystem { get; }
+
+		[Export ("expire")]
+		void Expire ();
+
+		[NullAllowed, Export ("contentProtectionSessionIdentifier")]
+		NSData ContentProtectionSessionIdentifier { get; }
+
+		[Export ("processContentKeyRequestWithIdentifier:initializationData:options:")]
+		void ProcessContentKeyRequest ([NullAllowed] NSObject identifier, [NullAllowed] NSData initializationData, [NullAllowed] NSDictionary<NSString, NSObject> options);
+
+		[Export ("renewExpiringResponseDataForContentKeyRequest:")]
+		void RenewExpiringResponseData (AVContentKeyRequest contentKeyRequest);
+	}
+
+	[TV (10,2), Mac (10,12,4), iOS (10,3)]
+	[BaseType (typeof (NSObject))]
+	interface AVContentKeyRequest {
+		[TV (10, 2), Mac (10, 12, 4), iOS (10, 3)]
+		[Field ("AVContentKeyRequestProtocolVersionsKey")]
+		NSString ProtocolVersions { get; }
+
+		[Export ("status")]
+		AVContentKeyRequestStatus Status { get; }
+
+		[NullAllowed, Export ("error")]
+		NSError Error { get; }
+
+		[NullAllowed, Export ("identifier")]
+		NSObject Identifier { get; }
+
+		[NullAllowed, Export ("initializationData")]
+		NSData InitializationData { get; }
+
+		[Export ("canProvidePersistableContentKey")]
+		bool CanProvidePersistableContentKey { get; }
+
+		[Export ("makeStreamingContentKeyRequestDataForApp:contentIdentifier:options:completionHandler:")]
+		void MakeStreamingContentKeyRequestData (NSData appIdentifier, [NullAllowed] NSData contentIdentifier, [NullAllowed] NSDictionary<NSString, NSObject> options, Action<NSData, NSError> handler);
+
+		[Export ("processContentKeyResponse:")]
+		void ProcessResponse (AVContentKeyResponse keyResponse);
+
+		[Export ("processContentKeyResponseError:")]
+		void ProcessError (NSError error);
+
+		[Export ("respondByRequestingPersistableContentKeyRequest")]
+		void RespondRequest ();
```

Comment:
`respondByRequesting` != `RespondRequest` the later sounds like "responding to a request" and not "responding by requesting`.

Another shorter name, like `RespondByRequesting` would remove the `Persistable` part which seems important - and including it only make sense with the rest of the `ContentKeyRequest`.

So it's a long name but `RespondByRequestingPersistableContentKeyRequest ()` might likely the safest way to bind it (as future releases might add similar API).